cmd/res/profiles/control-plane.profile.yaml

```diff
@@ -0,0 +1,29 @@
+name: "device-onvif-camera"
```

Comment:
Will it be better to name this with `control-plane` something? The two existing profiles in the profiles dir have pretty much same kind of naming.

Reply:
The reason we named it as such is because there are scripts and services that may need to filter out this device when querying for all devices (it gets returned as if it is a camera). If it was named as `control-plane` something, the name would need to be known by all of those scripts and services, and it would need to be not changed as to avoid breaking anything.

Because of this, we felt that naming it the same as the device service would allow anyone that knows the name of the service to know the name of the control-plane device.